### PR TITLE
A11y: Update buttons in Explore page to not use HTML Title for tooltip

### LINKED ISSUE #53888: A11y: Update buttons in Explore page to not use HTML Title for tooltip
**Steps to reproduce**:
1. Go here: https://play.grafana.org/explore
2. Hover over the "Split" button of the Toolbar. Hover over the icons buttons of the QueryOperationRowHeader ("Duplicate query", "Disable/enable query", "Remove query" and "Drag and drop to reorder").
3. After clicking on the "Split" button, hover on the "Close" one.

**Actual Result**:
Native HTML tooltip is displayed on **hover**, uses HTML title
No tooltip is displayed on **focus**

**Expected Result**
- Use of ARIA tags to ensure tooltip is accessible and correct styling applied to remain consistent with other tooltip implementations
- Tooltip displays on focus
**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
https://www.w3.org/WAI/WCAG22/Understanding/content-on-hover-or-focus

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

### PR DESCRIPTION
**Pull request by eledobleefe**
**What this PR does / why we need it**:
Use `tooltip` instead of `title` to fix a11y for these buttons in the Explore page.

**Which issue(s) this PR fixes**:

Fixes #53888 and it is related with #53844

**Special notes for your reviewer**:
Apart from the icon buttons of the Explore page ("Duplicate query" icon button in this screenshot for example):

ExplorePage_53888

we are fixing the ones of Transformation operations ("Debug" icon button in the screenshot for example):

TransformationOperations_53888

by modifying QueryOperationAction.tsx

### CODE REVIEW

File: public/app/features/explore/ExploreToolbar.tsx [hunk 1]
```diff
@@ -155,11 +155,11 @@ class UnConnectedExploreToolbar extends PureComponent<Props> {
         >
           <ToolbarButtonRow>
             {!splitted ? (
-              <ToolbarButton title="Split" onClick={() => split()} icon="columns" disabled={isLive}>
+              <ToolbarButton tooltip="Split the panel" onClick={() => split()} icon="columns" disabled={isLive}>
                 Split
               </ToolbarButton>
             ) : (
-              <ToolbarButton title="Close split pane" onClick={() => closeSplit(exploreId)} icon="times">
+              <ToolbarButton tooltip="Close split panel" onClick={() => closeSplit(exploreId)} icon="times">
```
Comment (Elfo404) [inline public/app/features/explore/ExploreToolbar.tsx:162]: it's actually pane, not panel :)

Comment (eledobleefe) [reply public/app/features/explore/ExploreToolbar.tsx:162]: 😅 thanks @Elfo404

File: public/app/features/explore/Wrapper.test.tsx [hunk 2]
```diff
@@ -211,17 +211,17 @@ describe('Wrapper', () => {
     });
   });
 
-  it('can close a pane from a split', async () => {
+  it('can close a panel from a split', async () => {
```
Comment (Elfo404) [inline public/app/features/explore/Wrapper.test.tsx:214]:
```suggestion
  it('can close a pane from a split', async () => {
```
😄

**Review by Elfo404 — APPROVED** [review]
🚀

**Pull request merged**